exercises/rational-numbers/RationalNumbersTest.cs

```diff
@@ -0,0 +1,240 @@
+// This file was auto-generated based on version 1.0.0 of the canonical data.
+
+using Xunit;
+
+public class RationalNumbersTest
+{
+    [Fact]
+    public void Add_two_positive_rational_numbers()
+    {
+        Assert.Equal(new RationalNumber (7, 6), new RationalNumber(1, 2).Add(new RationalNumber(2, 3)));
```

Comment:
I would prefer to only have one tested method: the operator. So could you remove this line? The same goes for the other tests.